src/components/Project/ProjectBasics/ProjectFormSidePanel/SaveIndicator.tsx

```diff
@@ -10,8 +10,22 @@ const SaveIndicator = () => {
   const project = useAppSelector(selectProject);
   const isSaving = useAppSelector(selectIsProjectSaving);
   const updatedMoment = moment(project?.updatedDate).format('D.M.YYYY HH:mm:ss');
+  const projectMode = useAppSelector(selectProjectMode);
 
-  return (
+  if (projectMode === 'new') {
+    return(
+    <div className="mt-4 flex justify-start">
+      <CustomTag
+        color={'var(--color-success)'}
+        text={isSaving ? 'Tallennetaan' : 'Ei tallennettu'}
+        weight={'light'}
+        textColor={'var(--color-white)'}
+        showLoading={isSaving}
+      />
+    </div>)
+  }
+  else {
+    return (
     <div className="mt-4 flex justify-start">
```

Comment:
This if else could be refactored as something like:

`const updateText = projectMode === "new" ? 'Ei tallennettu' : t('savedTime', { time: updatedMoment })

  return (
    <div className="mt-4 flex justify-start">
      <CustomTag
        color={'var(--color-success)'}
        text={isSaving ? 'Tallennetaan' : updateText}
        weight={'light'}
        textColor={'var(--color-white)'}
        showLoading={isSaving}
      />
    </div>
  )`
to avoid repetition. Also, should the 'Ei tallennettu' text come from translations as well?

Reply:
I made variables to translations, so now all all three comes from there: 'Ei tallennettu', 'Tallennetaan' and 'Viimeksi tallennettu'. Good proposal.